code/defines/procs/announcement.dm

```diff
@@ -47,9 +47,9 @@
 
 		switch(logging)
 			if(ARES_LOG_MAIN)
-				log_ares_announcement(title, message)
+				log_ares_announcement(title, message, signature)
 			if(ARES_LOG_SECURITY)
-				log_ares_security(title, message)
+				log_ares_security(title, message, signature)
```

Comment:
This should be replicated for `/proc/shipwide_ai_announcement(message, title = MAIN_AI_SYSTEM, sound_to_play = sound('sound/misc/interference.ogg'), signature, ares_logging = ARES_LOG_MAIN)` as well 
![image](https://github.com/cmss13-devs/cmss13/assets/76988376/f5052dbb-e742-4801-ad21-ad2d78ebec35)

Reply:
You also may consider updating all other usages of `log_ares_security` wherever applicable you have a user.